code/game/objects/items/nitrium_crystal.dm

```diff
@@ -0,0 +1,18 @@
+/obj/item/nitrium_crystal
+	desc = "A weird brown crystal, it smokes when broken"
+	name = "nitrium crystal"
+	icon = 'icons/obj/atmos.dmi'
+	icon_state = "nitrium_crystal"
+	var/cloud_size = 1
+
+/obj/item/nitrium_crystal/attack_self(mob/user)
+	. = ..()
+	var/datum/effect_system/smoke_spread/chem/smoke = new
+	var/turf/location = get_turf(src)
+	create_reagents(5)
+	reagents.add_reagent(/datum/reagent/nitrium_low_metabolization, 3)
+	reagents.add_reagent(/datum/reagent/nitrium_high_metabolization, 2)
+	smoke.attach(location)
```

Comment:
Could you explain the use case here?

Reply:
is a quick and dirty way to give a small boost or hide yourself, yes you go faster, but also you have to wait for the smoke to take effect and the smoke will make your pursuers faster too